Line/iOS/LINELoginDemo/LINELoginDemo/HUDView.h

```diff
@@ -0,0 +1,27 @@
+//
+//  HUDView.h
+//  LINELoginDemo
+//
+//  Created by Khanh LeViet on 10/7/16.
+//  Copyright (c) Google Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+#import <UIKit/UIKit.h>
+
+@interface HUDView : UIView
+
++ (instancetype)generateToView:(UIView *)view;
```

Comment:
Naming is mostly opinion, but I think this method should be called `addHUDViewToView:` since it's more descriptive of what the method does.